website/docs/r/cr_namespace.html.markdown

```diff
@@ -1,61 +1,39 @@
 ---
-
-subcategory: "Container Registry"
```

Comment:
please include this subcategory and remove sidebar_current

This changes we added recently for Terraform hashicorp registry docs

Reply:
We'll update the generator to address this.

Reply:
Done